docs/scos/dev/guidelines/testing-guidelines/executing-tests/test-the-glue-api.md

```diff
@@ -0,0 +1,395 @@
+---
+title: Test the Glue API
+description: How to test the Glue API
+template: howto-guide-template
+---
+
+This document describes how to set up and run Glue Backend API end-to-end tests. All code references the `WishlistsRestApi` module example and the `Pyz` project namespace. When you set up and run the tests for a different project namespace or module, adjust the names accordingly.
```

Comment:
* This guide explains how to set up and run Glue Backend API end-to-end tests using the WishlistsRestApi module as an example and the Pyz project namespace. Adjust the names accordingly for different project namespaces or modules.